README.md

```diff
@@ -17,7 +17,7 @@ This new Custom role along with the built-in "Reader" role is applied to the sub
 2. Open a Cloud Shell bash terminal (For details, see [Cloud shell docs](https://docs.microsoft.com/azure/cloud-shell/overview)).
 3. Run the following
 ```bash
-curl -s https://raw.githubusercontent.com/cloudconformity/temp-azure-custom-policy/master/apply-roles.sh | bash /dev/stdin \
+curl -s https://raw.githubusercontent.com/cloudconformity/temp-azure-custom-policy/master/apply-roles | bash /dev/stdin \
```

Comment:
I'm not a huge fan of encouraging people to stream a script directly from curl into bash (or any execution engine).

In this case the negative cases are pretty innocuous, but have seen it as a place where someone performing system admin via copy/paste could see some pretty nasty outcomes.

Consider the above personal opinion, feel free to ignore.